nix_update/__init__.py

```diff
@@ -68,6 +68,11 @@ def parse_args(args: list[str]) -> Options:
         help="Set filename where nix-update will update version/hash",
         default=None,
     )
+    parser.add_argument(
+        "--system",
+        help="The system used to to calculate the hash and run other nix commands",
+        default=None,
```

Comment:
Is this better?
 
```suggestion
        default="builtins.currentSystem",
```

Reply:
this wouldn't work when passed to `--system`, also things like `aarch64-linux` needs to be quoted